app/models/direct_assessment.rb

```diff
@@ -7,4 +7,26 @@ class DirectAssessment < ActiveRecord::Base
   has_many :results, as: :assessment
 
   validates :subject, presence: true
+  validate :ensure_single_department
+  before_save :set_department
```

Comment:
I feel like we had this discussion just a day or two ago... but with our validation now ensuring all associated outcomes are for the same department, do we still need assessments to have a direct reference to department? Or could we define `department` as `outcomes.first.department`?

Do we ever search for assessments by department?